src/backends/custom/custom_backend.cc

```diff
@@ -583,28 +583,24 @@ CustomBackend::Context::GetOutput(
     if (shape_dim_cnt > 0) {
       shape.assign(shape_dims, shape_dims + shape_dim_cnt);
     }
+
+    auto dst_memory_type = ToTRTServerMemoryType(*memory_type);
```

Comment:
`dst_memory_type` is not used anywhere, why change from directly using `ToTRTServerMemoryType(*memory_type)` in `AllocateOutputBuffer()`

Reply:
Fixed. Was using it to initialize actual_memory_type. Assigned directly instead.